app/components/manage-intermissions/edit.js

```diff
@@ -79,4 +83,49 @@ export default class manageIntermissionsEditComponent extends Component {
     yield intermission.destroyRecord();
     this.args.onClose();
   }
+
+  //position stuff
+  @tracked locationOptions = [
+    { code: 'during', name: 'Tijdens agendapunt' },
```

Comment:
I think this should use the i18n plugin to provide strings both in Dutch and English

Reply:
indeed, there's a `t` helper you can import and use